### PR TITLE
Spike: Merge changes in `rata` branch back to `develop`

### PR DESCRIPTION
**Pull request by ken-centrality**
## Why

`rata` branch currently contains code changes that cannot be merged back to `develop`. This PR aims to resolve this by consolidating constants by network and using `NEXT_PUBLIC_CENNZ_NETWORK` to determine their value

## What is changing

- Group relevant constants by network and have a single `process.env.NEXT_PUBLIC_CENNZ_NETWORK` to drive it
- Update API to `2.1.1-alpha.4`
- Fix API instance is prematurely disconnected causing a JS error
- Tie the default network mapping between CENNZ and Ethereum (Rata <-> Ropsten, Nikau <-> Kovan, Azalea <-> Mainnet)
- Reduce the wait time when polling for relayer status to 1 minute
- Move `tests` folder outside of `libs` folder (cc @aidan-starke)
- Set up `jest` to load env from `.env.test` instead of mocking `@/libs/constants` (cc @aidan-starke)

### CODE REVIEW

File: libs/constants.ts [hunk 1]
```diff
@@ -1,72 +1,96 @@
...
+	rata: {
+		ChainName: "Rata Testnet",
+		ChainId: {
+			InDec: 3000,
+			InHex: `0x${Number(3000).toString(16)}`,
+		},
+		ApiUrl: {
+			InWebSocket: "wss://rata.centrality.me/public/ws",
+		},
+		ExplorerUrl: "https://rata.uncoverexplorer.com",
```
Comment (aidan-starke) [inline libs/constants.ts:30]: Do we expect the need for different versions of `ApiUrl`?

File: libs/utils/generateGlobalProps.ts [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
 export interface GlobalProps {}
 
 export default async function generateGlobalProps(
 	section: SectionUri
 ): Promise<GlobalProps> {
-	const api = await Api.create({ provider: process.env.NEXT_PUBLIC_API_URL });
-
 	return {};
 }
```
Comment (aidan-starke) [inline libs/utils/generateGlobalProps.ts:11]: maybe delete this file, or use it to define props per page instead of calling Api.create() on each page?

**Review by aidan-starke — DISMISSED** [review]

File: libs/providers/CENNZApiProvider.tsx [hunk 3]
```diff
@@ -36,7 +38,7 @@ const CENNZApiProvider: FC<CENNZApiProviderProps> = ({
 		const api = initApi();
 
 		return () => {
-			void api.disconnect();
+			if (apiReady) void api.disconnect();
 		};
```
Comment (aidan-starke) [inline libs/providers/CENNZApiProvider.tsx:42]: can check api connected state from api directly - `if (api.isConnected) void api.disconnect();`

File: libs/constants.ts (hunk 1 above)
Comment (ken-centrality) [reply libs/constants.ts:30]: I thought we were using RPC too but doesn't seem to be, let's leave it for now, future-proofing.

File: libs/utils/generateGlobalProps.ts (hunk 2 above)
Comment (ken-centrality) [reply libs/utils/generateGlobalProps.ts:11]: good call, let's remove it

**Review by aidan-starke — APPROVED** [review]

**Pull request merged**